### DISCUSSION TITLE
Query could not be saved: postgres bd backend error

### DISCUSSION THREAD

**Discussion by KsM00000** [getredash/redash#6394] [Q&A] [answered]
Hi. I have redash version: 8.0.0+b32245 installed on Ubuntu server 18.04 via docker with postgres:9.6-alpine image as bd backend.
I have trouble with saving new queries and creating dashboards.
If I try to save query I get the error "query could not be saved".
![image](https://github.com/getredash/redash/assets/79855848/cdb3aab9-5e23-48ad-a196-abc407489744)
If I try to create dashboard window won't load.
![image](https://github.com/getredash/redash/assets/79855848/0f1e6e76-c4a8-47bf-a59f-46f0e5679dac)
I looked at the logs and saw the following errors.
```
ERROR:  duplicate key value violates unique constraint "users_pkey"
DETAIL:  Key (id)=(1) already exists.
STATEMENT:  UPDATE users SET updated_at=now() WHERE users.id = 1
```
After searching error I tried to reset users_id_seq with command
`SELECT setval(pg_get_serial_sequence('t1', 'id'), coalesce(max(id),0) + 1, false) FROM t1;`
Yet I still get the same error. 
I kinda get that I have an error with auto incrementing. The question is how to resolve it? I don't mind recreating the database (I have nothing but test data here anyway), but how to do it (I doesn't have a lot of experience with docker, honestly)?

**Comment by KsM00000** [comment, marked as answer]
Found error. If anybody is curious, there were duplicates in the table users that were causing errors. We deleted them and everything related to that user and everything is back to normal. To delete records manually:
docker exec -ti _postgres container id_ bash
psql -U postgres

**Reply by justinclift** [reply]
Cool.  You *might* need to set that sequence number to be higher than the current key id value as well, just to make sure you don't accidentally (attempt to) create create a future user with the same number. :smile:
